lib/InputTest.ts

```diff
@@ -749,6 +749,25 @@ export namespace InputTest {
  *
  * ---
  *
+ * Note: In Firefox, `deviceId` will be ignored, and instead the user will get a
+ * browser pop-up where they can select the device they want to use. This is
+ * unavoidable as it is Firefox's implementation of `getUserMedia()`.
+ *
+ * In most browsers, such as Chrome and Safari, when `getUserMedia()` is called,
+ * a prompt will ask the user for broad microphone-access permissions. Then, the
+ * parameters passed to `getUserMedia()` will determine the device that is
+ * captured.
+ *
+ * Firefox differs in that the prompt will ask for a specific input device.
+ * Regardless of the parameters passed to `getUserMedia()`, the device
+ * selected in that prompt will be captured. If the user opts to have the
+ * browser "Remember this selection" within the prompt, the device that was
+ * selected will be captured by every future `getUserMedia()` call as well.
+ * This selection will persist even through changes in the system OS, i.e. when
+ * default devices are changed.
```

Comment:
This looks good, as it describes the behavior depending on what you selected in the firefox popup window. Another question I have if I'm the customer, when this happens

If the user opts to have the browser "Remember this selection" within the prompt, the device that was selected will be used for every future `gUM` call as well.

How do I undo this? How do I select a different device because after selecting "Remember this selection", I'm not able to change the device anymore.

Reply:
I think we can just add a line that points to this link https://support.mozilla.org/en-US/kb/how-manage-your-camera-and-microphone-permissions

> See this [page](https://support.mozilla.org/en-US/kb/how-manage-your-camera-and-microphone-permissions) for more information on managing microphone permission with Firefox